analyzers/its/regression-test.ps1

```diff
@@ -33,6 +33,9 @@ function Prepare-Project([string]$ProjectName){
     $Output = ".\output\$ProjectName"
     New-Item -ItemType directory -Path $Output | out-null
 
+    Write-Output "Copying `Directory.Build.targets` to the project directory ($ProjectName)..."
+    Copy-Item .\Directory.Build.targets -Destination $ProjectName
```

Comment:
Why is this even needed? If we drop the file in `analyzers\its` or `analyzers\its\sources`, it should be automatically picked up by the builds. Shouldn't it?

Reply:
The file is already in `analyzers\its`. 

According to [the search scope paragraph](https://learn.microsoft.com/en-us/visualstudio/msbuild/customize-by-directory?view=vs-2022#search-scope) it will look all the way to the root so indeed, we can remove this.

I will do a PR to simplify this even more. I deleted quite a lot of lines of code by renaming a file 😆